client/store/admin.js

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios'
+
+//ACTION TYPES
+const GOT_USERS = 'GOT_USERS'
+const GOT_SINGLE_USER = 'GOT_SINGLE_USER'
+
+//ACTION CREATORS
+export const gotUsers = users => ({
+  type: GOT_USERS,
+  users
+})
+
+export const gotSingleUser = user => ({
+  type: GOT_SINGLE_USER,
+  user
+})
+
+//THUNK CREATORS
+export const fetchUsers = () => {
+  return async dispatch => {
+    try {
+      const {data} = await axios.get('/api/admin/users')
+      console.log(data)
```

Comment:
comment this out or remove? Just so it doesnt log during our presentation tomorrow